.github/workflows/build.yml

```diff
@@ -54,6 +54,12 @@ jobs:
             && (echo 'solr is running: pass' && exit 0)
             || (echo 'solr is running: fail' && exit 1)
 
+      - name: Test request to port 8983.
+        run: >
+            curl -f http://localhost:8983/
+            && (echo 'Request to port 8983: pass' && exit 0)
+            || (echo 'Request to port 8983: fail' && exit 1)
```

Comment:
I attempted to test this check on my computer but I couldn't set it up correctly, if it's too silly let me know.